packages/docs-site/src/content/docs/docs/guides/env-files.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: How DMNO uses .env files
+---
+
+DMNO uses `.env` files in two ways:
+1. To scaffold out the initial [`schema`](/docs/guides/schema/#defining-config-items) in your `config.mts` files when your run `dmno init`
+2. To set value overrides to be used while resolving your config values
+
+Let's zoom in on each of these.
+
+## Scaffolding the schema
+
+When you run `dmno init`, DMNO will look for all `.env` files in your project and use them to scaffold out the `schema` in your `config.mts` files. This includes _all_ `.env` files, regardless of being gitignored or checked in, including samples, and environment specific files.
+
+We try to infer as much about each config item as possible by:
+- Including related comments as a `description`
+- Infer the type (`extends`) based on the value for basic types like `boolean`, `number`, `email`, `url`
+- Set the `value` and not mark it as `sensitive` if the file was checked into source control. This includes using `value: switchByNodeEnv(...)` if values were found in multiple environment specific files
+- Set an `exampleValue` from a value from a `.env.sample`
+
+This automatic scaffolding of your config `schema` is meant to be a good starting point, but you should review it make adjustments as necessary.
+
+:::note
+`dmno init` will also prompt you to delete the `.env` files that were previously checked into source control because the values in those files have now been incorporated into your new config schema
```

Comment:
```suggestion
`dmno init` will also prompt you to delete the `.env` files that were previously checked into source control because the values in those files have now been incorporated into your new config schema.
```